handlers.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aleksl0l/bomb-backend/message"
+	"github.com/aleksl0l/bomb-backend/player"
+	"github.com/golang/protobuf/proto"
+	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+)
+
+func serveWs(w http.ResponseWriter, r *http.Request) {
+	conn, err := upgrader.Upgrade(w, r, nil)
+	if err != nil {
+		return
+	}
+
+	_, data, err := conn.ReadMessage()
+	if err != nil {
+		return
+	}
+	msg := &message.Message{}
+	err = proto.Unmarshal(data, msg)
+	if err != nil {
+		return
+	}
+	switch msg.Type {
+	case message.MessageType_JOIN_GAME:
```

Comment:
Maybe implement such kind of router instead of this case? 

I mean, create struct with pairs of possible message types and separate handlers from WS serving and creating logic.

Reply:
It will be helpful if we'll be adding new message handlers in future.

Reply:
if we'll want a new message handler in the future, we need just add a new type in the protobuf file and add a handler for it

Reply:
Yeah, I mean extract this switch into hashmap of functions.